cmake/Modules/FindSDL_mixer.cmake

```diff
@@ -0,0 +1,110 @@
+#.rst:
```

Comment:
I don't think this is a good idea. Can't we just use a more recent version of CMake? 3.5 would be my minimum recommendation but 3.8 would be better as it adds `target_compile_features(... cxx_14)`

Reply:
yea 3.6 is in backports, so should be not a problem. I only wanted to fix it first before changing a lot of versions. We could leave the file out in the end.

Reply:
Would be easier to do earlier than later. There might be more bugs in a x.0 version ;)